src/SConscript

```diff
@@ -30,36 +32,54 @@ def generate_daos_version_header(env):
     out = env.Substfile(tmpl_hdr_in, SUBST_DICT=sd)
     print("generated daos version header file: {}".format(out[0].abspath))
 
+def check_header(env, header):
+    """Check a header file"""
```

Comment:
This does more than check a header file though doesn't it?

Reply:
It just makes sure we try to build the header file (though it could be used for more than headers if that's what you mean)

Reply:
I mean the comment says the function does one thing but it does two and should probably do three.